src/components/TimePicker/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {ReactElement} from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import {Picker} from '@react-native-picker/picker';
+import {Control, Controller} from 'react-hook-form';
+import {CreateQuestionFieldType} from '@src/screens/CreateQuiz/components/CreateQuestion/CreateQuestion';
+import {TextError} from '@src/components/ui/ReadyStyles/TextError';
+
+type TimePickerPropsType = {
+  control: Control<CreateQuestionFieldType>;
+  errors: string | undefined;
+};
+
+export const TimePicker = ({control, errors}: TimePickerPropsType) => {
+  const getMinutesItems = () => {
+    const items: ReactElement[] = [];
+    for (let i = 0; i <= 59; i++) {
+      items.push(<Picker.Item testID="minutesItem" key={i} value={i} label={String(i)} />);
+    }
+    return items;
+  };
+
+  const getSecondsItems = () => {
+    const items: ReactElement[] = [];
+    for (let i = 0; i <= 59; i += 5) {
+      items.push(<Picker.Item testID="minutesItem" key={i} value={i} label={String(i)} />);
+    }
+    return items;
+  };
```

Comment:
lets separate this into component with prop. something like : `<PickerItems type='seconds|minutes'>`